config/main.py

```diff
@@ -1999,6 +2002,22 @@ def override_config_db(config_db, config_input):
     click.echo("Overriding completed. No service is restarted.")
 
 
+def table_hard_dependency_check(config_json):
+    aaa_table_hard_dependency_check(config_json)
+
+
+def aaa_table_hard_dependency_check(config_json):
+    AAA_TABLE = config_json.get("AAA", {})
+    TACPLUS_TABLE = config_json.get("TACPLUS", {})
+
+    aaa_authorization_login = AAA_TABLE.get("authorization", {}).get("login", "")
```

Comment:
I suggest use authentication, because some device may not enable authorization but always will enable authentication.

Reply:
Do you suggest to add authentication? This is a fix item specific for authorization failure from our ADO: 26205194

Reply:
This PR is for 26371181: "Improve load_minigraph command, add pre-check in load minigraph, if passkey is empty and TACACS enabled, load minigraph should failed."

The reason is 'tacacs enabled' means following scenarios:
1. authentication enabled but authorization not enabled.
2. authentication and authorization both enabled.

current code can't cover scenario 1.